st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -1,272 +0,0 @@
-# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
Yes, I like this. I also think I will do this for the test runners. This is much cleaner than throwing symlinks around everywhere.